storage/myisam/mi_check.c

```diff
@@ -1,5 +1,6 @@
 /*
    Copyright (c) 2000, 2016, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2018, Percona and/or its affiliates. All rights reserved.
```

Comment:
Should (C) be for Percona, MariaDB, or both here?

Reply:
Fixed. Put both.